transit_odp/site_admin/tests/test_exports.py

```diff
@@ -196,13 +207,27 @@ def test_active_publisher_to_string_last_four_columns(self, client_factory):
             granularity=PPCReportType.WEEKLY,
             created=today - timedelta(days=7),
         )
+        PostPublishingCheckReportFactory(
+            dataset=dataset4,
+            vehicle_activities_analysed=0,
+            vehicle_activities_completely_matching=0,
+            granularity=PPCReportType.WEEKLY,
+            created=today,
+        )
+        PostPublishingCheckReportFactory(
+            dataset=dataset5,
+            vehicle_activities_analysed=0,
+            vehicle_activities_completely_matching=0,
+            granularity=PPCReportType.WEEKLY,
+            created=today,
+        )
```

Comment:
The correct check for "undergoing validation" is that no weekly PPC report has been generated yet. It's probably a grey area whether a report with 0 vehicles analysed constitutes "undergoing validation" - maybe it does. Anyway, better to just not generate a report for this dataset.

Reply:
I don't see how this has been resolved.

Reply:
I removed the granularity that is a constraints